csharp/src/IceDiscovery/PluginI.cs

```diff
@@ -79,12 +79,12 @@ public void initialize()
                 properties.setProperty("IceDiscovery.Locator.AdapterId", Guid.NewGuid().ToString());
             }
 
-            _multicastAdapter = _communicator.createObjectAdapter("IceDiscovery.Multicast");
-            _replyAdapter = _communicator.createObjectAdapter("IceDiscovery.Reply");
-            _locatorAdapter = _communicator.createObjectAdapter("IceDiscovery.Locator");
+            _multicastAdapter = _communicator.createObjectAdapter("IceDiscovery.Multicast", authenticationOptions: null);
```

Comment:
It's not clear why you specify the default value explicitly here.

Reply:
When I first added the method I didn't add a default value to see where it was used. We can now use the default.